### PR TITLE
Remove VT1100 and VT1300 Libraries

### PR DESCRIPTION
**Pull request by VertorixAU**
I am the creator of the libraries and wish for them to be removed.
Thank you

### CODE REVIEW

**Comment by per1234** [comment]
Hello @VertorixAU. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Comment by umbynos** [comment]
Done:
```
022/08/31 16:00:43 Loaded 5126 libraries from DB
Removing VT1100_Mini
warning: Library Git clone folder /home/ubuntu/workspace/gitclones/github.com/VertorixAU/Vertorix_VT1100_Mini not present
Removing VT1300_Shield
warning: Library Git clone folder /home/ubuntu/workspace/gitclones/github.com/VertorixAU/Vertorix_VT1300_Shield not present
Success!
```
The warning should be expected since the libraries are no longer publicly available on GitHub

**Review by umbynos — APPROVED** [review]

**Pull request merged**
